python/python/res/job_queue/ext_job.py

```diff
@@ -137,6 +143,31 @@ def get_max_running_minutes(self):
     def set_max_running_minutes(self, min_value):
         self._set_max_running_minutes(min_value)
 
+    def minimumArgumentCount(self):
+        return self._min_arg()
+
+    def maximumArgumentCount(self):
+        return self._max_arg()
+
+    def argumentTypes(self):
+        """ @rtype: list of type """
+
+        result = []
+        for index in range(self.maximumArgumentCount()):
+            t = self._arg_type(index)
+            if t == ContentTypeEnum.CONFIG_BOOL:
+                result.append(bool)
+            elif t == ContentTypeEnum.CONFIG_FLOAT:
+                result.append(float)
+            elif t == ContentTypeEnum.CONFIG_INT:
+                result.append(int)
+            elif t == ContentTypeEnum.CONFIG_STRING:
+                result.append(str)
+            else:
+                result.append(None)
```

Comment:
`None` is not a type, and I would expect type here.  What about using `NoneType` if that is actually the meaning of `None`?

Reply:
Thanks. I will take these comments into account in another pull request.